plugins/linear-genome-view/src/LinearGenomeView/components/OverviewScaleBar.tsx

```diff
@@ -33,12 +35,58 @@ const useStyles = makeStyles(theme => {
       backgroundColor: theme.palette.background.default,
       position: 'relative',
       borderColor: theme.palette.text.primary,
+      borderBottomColor: 'black',
+    },
+    scaleBarContigForward: {
```

Comment:
Do these presume a specific height for the header bar?

Reply:
They take into account the header overview height. If the the header overview height increases, the pattern will just space out a tad more.